### DISCUSSION TITLE
TS0601_soil sensor issue

### DISCUSSION THREAD

**Discussion by jturello-ulink** [Koenkk/zigbee2mqtt#23740] [Devices]
Hello,

I've recently bought a Tuya Rehent moisture soil sensor and after using it successfully for a few weeks with Tuya, I switched to Z2M running as module on  my Home Assistant (Rapsberry 3 + SonOFF version E Dongle with latest ember firmware).

It worked for some times, but suddenly temperature value became too high (like 35°C while external temp was 20°C), and even temperature changes weren't realistic: increase at night while external temp was going down. And now it only increase and it's currently at 54°C!

The main changes that occured (can't remember when) was and addition of a second SonOff dongle with router firmware to extend network, so I tough that maybe the increase of received values frequency was due to better signal reception (was a little low like 40-50RQI before). 

Any idea? What can I do, reset my soil sensor?

Thanks!
![Capture d’écran du 2024-08-26 11-18-29](https://github.com/user-attachments/assets/c379a17e-0994-4daa-855a-c2b4177a800e)

**Comment by pubFbx** [comment]
Hello, same problem: temperature=54°C.
Have you found a solution?
